packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -82,13 +86,23 @@ function useMovingAnimation( {
 			}
 		}
 
-		if ( prefersReducedMotion ) {
+		if (
+			window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches ||
+			isTyping() ||
+			getGlobalBlockCount() > BLOCK_ANIMATION_THRESHOLD
+		) {
 			// If the animation is disabled and the scroll needs to be adjusted,
 			// just move directly to the final scroll position.
 			preserveScrollPosition();
 			return;
 		}
 
+		const isPartOfSelection =
+			isSelected ||
+			isBlockMultiSelected( clientId ) ||
+			isAncestorMultiSelected( clientId );
+		const zIndex = isPartOfSelection ? '1' : '';
```

Comment:
It's unclear as well why we're doing this. (Higher z-index for the selected blocks I guess)

Reply:
Ok will add a comment. I think it's so the other block goes "under" the selected blocks.